recipes/kcov/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class KcovConan(ConanFile):
+    name = "kcov"
+    license = "GPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index/"
+    homepage = "http://simonkagstrom.github.io/kcov/index.html"
+    description = "Code coverage tool for compiled programs, Python and Bash\
+        which uses debugging information to collect and report data without\
+            special compilation options"
+    topics = ("coverage", "linux", "debug")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "CMakeLists.txt", "patches/**"
+    requires = ["zlib/1.2.11",
+                "libiberty/9.1.0",
+                "libcurl/7.64.1",
+                "elfutils/0.180"]
+    generators = "cmake"
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
```

Comment:
Can you build using mingw on Windows?

Reply:
I'm not the author, i'll ask him.
first line of readme.md  says : _"Kcov is a FreeBSD/Linux/OSX code coverage tester"_  .

Reply:
Thanks, so can drop Windows if it is the case.

Reply:
ok i do it